lib/sunomono/locales/pt.yml

```diff
@@ -7,6 +7,29 @@ pt:
     insert_steps: "Insira os passos"
     elements: "Declare todos os elementos da tela"
     actions: "Declare todas as acoes da tela"
+    wait_1: "aguardar_progresso é um metodo da classe base, então não importa o que é"
```

Comment:
You only added locales for pt, so the gem will only work for PT projects.
This is not an improvement, is a regression, because today we can create an English project, if we merge your PR, we will only create Portuguese projects